src/main/java/core/basesyntax/Plane.java

```diff
@@ -1,11 +1,73 @@
 package core.basesyntax;
 
 /**
- * <p>Дано класс Plane. Написать класс PlaneBuilder, который реализует паттерн Builder.
+ * <p>Дано класс Plane. Написать класс PlaneBuilder, который реализует паттерн PlaneBuilder.
  * Добавьте не менее пяти полей в класс Plain. Внести все необходимые изминения в класс Plane
- * которые могут потребоваться для реализации паттерна Builder.</p>
+ * которые могут потребоваться для реализации паттерна PlaneBuilder.</p>
  */
 
 public class Plane {
+    private String model;
+    private int numberOfEngines;
+    private int maxDistance;
+    private String company;
+    private int year;
 
+    public Plane(String model, int numberOfEngines, int maxDistance, String company, int year) {
+        this.model = model;
+        this.numberOfEngines = numberOfEngines;
+        this.maxDistance = maxDistance;
+        this.company = company;
+        this.year = year;
+    }
+
+    private Plane(PlaneBuilder planeBuilder) {
+        this.model = model;
+        this.numberOfEngines = numberOfEngines;
+        this.maxDistance = maxDistance;
+        this.company = company;
+        this.year = year;
+    }
+
+    public static class PlaneBuilder {
+        private String builderName;
+        private String model;
+        private int numberOfEngines;
+        private int maxDistance;
+        private String company;
+        private int year;
+
+        public PlaneBuilder(String builderName) {
```

Comment:
```suggestion
        public PlaneBuilder() {
```
create setter for  builderName

Reply:
@Sofasmile 
Just a regular setter, that we can create using "Generate" option in IDEA ?